tests/cppunit/cluster_test.cc

```diff
@@ -113,8 +113,8 @@ TEST(Cluster, CluseterGetNodes) {
 
   std::vector<std::string> vnodes = Util::Split(output_nodes, "\n");
 
-  for (unsigned i = 0; i < vnodes.size(); i++) {
-    std::vector<std::string> node_fields = Util::Split(vnodes[i], " ");
+  for (const auto& vnode : vnodes) {
```

Comment:
Just a tiny note here: move `&` close to the `vnode` variable, not the type.

Reply:
done